src/core_functions/aggregate/distributive/bitstring_agg.cpp

```diff
@@ -43,6 +45,21 @@ struct BitstringAggBindData : public FunctionData {
 		}
 		return false;
 	}
+
+	static void Serialize(Serializer &serializer, const optional_ptr<FunctionData> bind_data_p,
+	                      const AggregateFunction &) {
+		auto &bind_data = bind_data_p->Cast<BitstringAggBindData>();
+		serializer.WriteProperty(100, "min", bind_data.min);
+		serializer.WriteProperty(101, "max", bind_data.max);
+	}
+
+	static unique_ptr<FunctionData> Deserialize(Deserializer &deserializer, AggregateFunction &) {
+		Value min;
+		Value max;
+		deserializer.ReadProperty(100, "min", min);
+		deserializer.ReadProperty(101, "max", max);
```

Comment:
I'm not sure what the right way here is to guarantee bwc, so that this can handle plans serialized from an older duckdb version.